keps/sig-node/4540-strict-cpu-reservation/README.md

```diff
@@ -411,7 +405,9 @@ Longer term, we may want to require automated upgrade/rollback tests, but we
 are missing a bunch of machinery and tooling and can't do that now.
 -->
 
-We manually test it in our internal environment and it works.
+End users have this feature enabled in v1.32 under `CPUManagerPolicyAlphaOptions` (default to false) will continue to have the feature enabled in v1.33 under `CPUManagerPolicyBetaOptions` (default to true) automatically i.e. no extra action is needed.
+
+To enable or disable this feature in v1.33, follow the feature activation and de-activation procedures described above.
```

Comment:
My expectation for this is roughly something along these lines: https://github.com/kubernetes/enhancements/blob/master/keps/sig-apps/3850-backoff-limits-per-index-for-indexed-jobs/README.md#were-upgrade-and-rollback-tested-was-the-upgrade-downgrade-upgrade-path-tested

Reply:
Ok, I will re-write following the example.